README.md

```diff
@@ -1,60 +1,81 @@
 # SuperTuxKart
+
 [![Build Status](https://travis-ci.org/supertuxkart/stk-code.svg?branch=master)](https://travis-ci.org/supertuxkart/stk-code)
-[![AppVeyor Build Satus](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
-[![#supertuxkart on freenode](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
+[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
 
 SuperTuxKart is a free kart racing game. It focuses on fun and not on realistic kart physics. Instructions can be found on the in-game help page.
 
 The SuperTuxKart homepage can be found at <https://supertuxkart.net/>. There is also our [FAQ](https://supertuxkart.net/FAQ) and information on how get in touch with the [community](https://supertuxkart.net/Community)
 
-## Hardware Requirements
-* You need a 3D graphics card. (NVIDIA GeForce 8xxx and higher, ATI Radeon HD 4xxx and higher or Intel HD 3000 and higher.)
-* You should have a CPU that's running at 1GHz or better.
-* You'll need at least 512 MB of free VRAM (video memory).
-* Disk space: 400MB
-* Ideally, you want a joystick with at least 6 buttons.
+## System recommendations for running SuperTuxKart
+
+To run SuperTuxKart, make sure that your computer's specifications are equal or higher than the following recommended specifications:
+
+### Desktop
+
+To run SuperTuxKart on your desktop PC, check that your system specifications are equal to, or newer than the following system specifications:
+
+* CPU: Intel Pentium G2020 or AMD A4-4000 (and newer)
+* Graphics Card: NVIDIA GeForce GT 520 or AMD Radeon HD 6450, or Intel HD Graphics (and newer) - OpenGL >=3.1
+* System memory: 4 GB
+* Storage: 800 MB
+* Monitor resolution: 1024*768 (and higher)
+* Mouse and keyboard (a gamepad is recommended)
+
+### Laptop
+
+To run SuperTuxKart with the Antarctica engine on your laptop, check that your laptop's specifications are equal to, or higher than these recommendations:
+
+* Manufacture date: Newer than September 2013
```

Comment:
Not very specific, my ol’ laptop from 2010 can still run stk at very low specs, sure but it already wasnt a war machine back in the days.
Why simply not drop that line ?